hydra_agent/hydra_graph.py

```diff
@@ -55,7 +55,6 @@ def get_endpoints(self,api_doc, redis_connection):
             alias="Entrypoint",
             properties=entrypoint_properties)
         self.redis_graph.add_node(entrypoint_node)
-        redis_connection.set("EntryPoint", entrypoint_properties)
```

Comment:
@ShravanDoda `classes_objects` and `collection-endpoints` also have the same error. 
I think @PrajwalM2212 also working on the same issue in PR #73. So, Please ask him.